### ISSUE TITLE
Shortcode in Contact Form 7 contains two inputs within one label

### ISSUE THREAD

**Issue by msroberts** [ibericode/mailchimp-for-wordpress#361]
```html
<div>
<!-- MailChimp for WordPress v4.0.2 - https://mc4wp.com/ -->
<p class="mc4wp-checkbox mc4wp-checkbox-contact-form-7">
<label>
<input type="hidden" name="_mc4wp_subscribe_contact-form-7" value="0" />
<input type="checkbox" name="_mc4wp_subscribe_contact-form-7" value="1" checked="checked" />
<span>Sign me up for the newsletter!</span>
</label>
</p>
<!-- / MailChimp for WordPress -->
</div>
```

This is causing accessibility issues in some browsers, which don't let the user check the field by clicking the label text.

Any chance the hidden field could be placed outside the `label` element?

**Comment by dannyvankooten** [comment]
Hey @msroberts ,

Nice catch, in which browsers did this occur for you? Consider this fixed in the next plugin update, thanks for the bug report!

**Comment by msroberts** [comment]
It happens in all browsers (the HTML is the same).  I only noticed a problem checking the box in IE (Chrome was able to figure out which `input` to select).

**Comment by msroberts** [comment]
Looks like it's fixed in v4.0.5.

**Comment by dannyvankooten** [comment]
Hi @msroberts ,

Definitely, I took your input and included this in last week's release. Hope that helps!
